pkg/controllers/user/helm/common/common.go

```diff
@@ -160,6 +162,20 @@ func InstallCharts(tempDirs *HelmPath, port string, obj *v3.App) error {
 	return nil
 }
 
+// getTimeoutArgs generates the appropriate arguments for helm depending on the app's timeout and wait fields
+func getTimeoutArgs(app *v3.App) []string {
+	var timeoutArgs []string
+
+	if app.Spec.Wait {
+		timeoutArgs = append(timeoutArgs, "--wait")
+	}
+	// the default timeout in helm is 300 seconds
+	if timeout := app.Spec.Timeout; timeout != 300 {
```

Comment:
I think we can just append this regardless what the value is. This would be useful for debugging the commands to know what the expected timeout is.